src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/SimpleRenameStrategy.cs

```diff
@@ -23,7 +23,7 @@ public override bool CanHandleRename(string oldFileName, string newFileName)
         {
             var oldNameBase = Path.GetFileNameWithoutExtension(oldFileName);
             var newNameBase = Path.GetFileNameWithoutExtension(newFileName);
-            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase);
+            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase) && !Equals(Path.GetFileName(oldNameBase), Path.GetFileName(newNameBase));
         }
```

Comment:
@davkean , i am comparing the filename's not path . Have switched to using string.Compare.
I compared this with VS 14 and the rename is case sensitive. So rename is triggered if you change from Foo to foo. Let me know if you think otherwise